### ISSUE TITLE
Why is editing access blocked on 'Fundraising on Giveth' doc?

### ISSUE THREAD

**Issue by clara-zi** [Giveth/giveth-planning#671]
@Cotabe

**Comment by Cotabe** [comment]
Yes @clara-zi . I can't edit. I don't know if I am doing something wrong :(

**Comment by clara-zi** [comment]
@Cotabe Access is showing as being open to everyone in Giveth? Do you have access to edit the Notion workspace normally? Can you edit using this link? https://giveth.notion.site/Your-funds-a-guide-for-projects-af4cd4114eb94caca27b18bbf24ee73f

**Comment by ahmadabugosh** [comment]
@Cotabe just reshared the invite (check your email)

**Comment by Cotabe** [comment]
Got access. Thanks @ahmadabugosh and @clara-zi 

Just edited and posted the offramping for Mexico https://www.notion.so/giveth/Mexico-d089d0835203451fa6fdc94d5e779e48

And linked to it some Spanish speaking countries to that one in case it can be useful. Example: https://www.notion.so/giveth/Chile-321377b03d1644999760119a92a2d0fa

**Comment by clara-zi** [comment]
Thanks @Cotabe. Closing now
